resources/minegrub-update.service

```diff
@@ -2,7 +2,7 @@
 Description=minegrub theme splash update service
```

Comment:
Something like "minegrub splash text and package count update service" would be more descriptive with the new feature :>